sandbox/src/layer-manager/layer-manager.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import ReactMapGL from 'react-map-gl'
+import styles from './layer-manager.module.css'
+
+import LayerManager from '@globalfishingwatch/map-components/src/layer-manager'
+
+class MapPage extends Component {
+  state = {
+    mounted: false,
+    viewport: {
+      width: '100%',
+      height: '100%',
+      latitude: 37.7577,
+      longitude: -122.4376,
+      zoom: 8
+    },
+    layers: [
+      { id: 'north-star', type: 'basemap' },
+      {
+        id: 'mpant',
+        type: 'carto',
+        color: '#58CFFF',
+        opacity: 1,
+      },
+      {
+        id: 'eez',
+        type: 'carto',
+        color: '#A9ACFF',
+        opacity: 1,
+      },
+      {
+        id: 'bluefin_rfmo',
+        type: 'carto',
+        color: '#B3CF9F',
+        opacity: 1,
+      },
+      {
+        id: 'cp_rfmo',
+        type: 'carto',
+        color: '#58CFFF',
+        opacity: 1,
+        selectedFeatures: {
+          field: 'rfb',
+          values: ['IATTC'],
+          color: '#58CFFF',
+          style: {
```

Comment:
A suggestion here: 
`selectedFeatures` was meant to be extremely flexible and generic, thus allowing completely custom style bits for selected features (because it's also supposed to work with data portal's event layers).
I'm not sure we need that much flexibility here. I would simplify the API to cover the use cases we actually need now (so in map-client and carrier portal)